src/commands/EventCommand.js

```diff
@@ -70,126 +70,143 @@ class EventCommand extends SubsCommand {
-                        return "command.event.wrong-date-format";
-                    else
-                        return true;
-                },
-                "value": (content, values) => {
-                    if (content == "-")
-                        return values["start"];
-                    else
-                        return content;
-                }
-            },
-            {
-                "name": "role",
-                "example": "member",
-                "validate": (content) => {
-                    if (!this.eventModule.isMentionableRole(content)) 
-                        return ["command.event.role-not-valid", this.eventModule.getMentionableRoles().join(", ")];
-                    else
-                        return true;
-                }
-            },
-            {
-                "name": "place",
-                "example": ["Škola", "School", "Arbesovo náměstí"],
-                "validate": (content) => {
+        const builder = new CommandBuilder("event.create", message.author, channel, [{
+            "name": "type",
+            "example": types,
+            "validate": (content) => {
+                if (!types.includes(content))
+                    return ["command.event.type-not-valid", types.join(", ")];
+                else
                     return true;
-                }
-            },
-            {
-                "name": "subject",
-                "example": ["STP", "?"],
-                "validate": (content) => {
+            }
+        },
+        {
+            "name": "title",
+            "example": Translation.translate("builder.event.create.title.example"),
+            "validate": (content) => {
+                return true;
+            }
+        },
+        {
+            "name": "name",
+            "example": Translation.translate("builder.event.create.name.example").split(","),
+            "validate": (content) => {
+                if (this.eventModule.exists(content))
+                    return "command.event.already-exists";
+                else
                     return true;
-                }
+
             },
-            {
-                "name": "description",
-                "example": ["Popis eventu", "Description of event"],
-                "validate": (content) => {
+            "value": (content, values) => {
+                if (content == "-")
+                    return values["title"].toLowerCase().split("").map(function(letter) {
+                        const i = this.accents.indexOf(letter);
+                        return (i !== -1) ? this.acc_out[i] : letter;
+                    }.bind({
+                        accents: "àáâãäåąßòóôőõöøďdžěèéêëęðçčćìíîïùűúûüůľĺłňñńŕřšśťÿýžżźž- ",
+                        acceOut: "aaaaaaasoooooooddzeeeeeeeccciiiiuuuuuulllnnnrrsstyyzzzz__"
+                    })).join("");
+            }
```

Comment:
This method changes the value. You need to return content; if content is  not -.
```suggestion
                else
                    return content;
            }
```